### ISSUE TITLE
[Bug] Accessibility: Unlabeled buttons in mail reader

### ISSUE THREAD

**Issue by davidbtc2009** [ProtonMail/proton-mail-android#177]
**Describe the bug**
In the reader view once an email is opened the button to see more details about who sent a message and at what time is unlabeled. The buttons in the bottom row (reply, mark unread, the tag icon, more 3 dot button) also have no labels when talkback is used

**To Reproduce**
1. Enable talkback from system settings -> Accessibility -> Talkback -> Use Talkback
2. Open Proton Mail app
3. Double tap an email to open it
5. Tap the down arrow icon to the right of the sender information and the icons on the bottom row

**Expected behavior**
The icons should be labeled so talkback users know what they do

**Smartphone (please complete the following information):**
 - Device: Motorola One 5G Ace
 - OS: Android 11
 - Version 3.0.3
